app/src/Model/Table/GatheringsTable.php

```
@@ -224,4 +227,88 @@ public function findByType(\Cake\ORM\Query\SelectQuery $query, array $options):
+     * After save callback
+     *
+     * Syncs template activities from the gathering type when:
+     * 1. A new gathering is created
+     * 2. The gathering type is changed
+     *
+     * @param \Cake\Event\EventInterface $event The event object
+     * @param \Cake\Datasource\EntityInterface $entity The saved entity
+     * @param \ArrayObject $options Options passed to save
+     * @return void
+     */
+    public function afterSave(EventInterface $event, EntityInterface $entity, \ArrayObject $options): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event'. (undefined)

(UnusedFormalParameter)